README.md

```diff
@@ -318,6 +318,27 @@ See [#158](https://github.com/remarkablemark/html-react-parser/issues/158).
 
 Enable the [trim](https://github.com/remarkablemark/html-react-parser#trim) option. See [#155](https://github.com/remarkablemark/html-react-parser/issues/155).
 
+#### Don't change case of tags.
```

Comment:
[Preview](https://github.com/remarkablemark/html-react-parser/blob/dc543d145e74acf4f88be9c97fbe3a600a6d8d7f/README.md#dont-change-case-of-tags)